data/json/items/book/maps.json

```diff
@@ -201,5 +201,131 @@
+        "bridge",
+        "road",
+        "s_grocery",
+        "trailer_commercial",
+        "bakery"
+      ],
+      "message": "You add roads and local grocery and convenience stores as well as bakeries to your map."
+    }
+  },
+  {
+    "id": "historybrochure",
+    "copy-from": "abstractmap",
+    "type": "GENERIC",
+    "name": { "str": "history advertising brochure" },
+    "description": "This is a small brochure advertising for local museums, libraries and antique stores with an address below each entry. Using it will add roads an local museums, libraries and antique stores to your map.",
+    "color": "light_blue",
+    "use_action": {
+      "type": "reveal_map",
+      "radius": 90,
+      "terrain": [
+        "bridge",
+        "road",
+        { "om_terrain": "museum", "om_terrain_match_type": "TYPE" },
+        "s_antique",
+        "s_library"
+      ],
+      "message": "You add roads and local museums, libraries and antique stores to your map."
+    }
+  },
+  {
+    "id": "mallbrochure",
+    "copy-from": "abstractmap",
+    "type": "GENERIC",
+    "name": { "str": "mall advertising brochure" },
+    "description": "This is a small brochure advertising for local malls with an address below each entry. Using it will add roads an local malls, strip malls and megastores to your map.",
+    "color": "light_blue",
+    "use_action": {
+      "type": "reveal_map",
+      "radius": 90,
+      "terrain": [
+        "bridge",
+        "road",
+        "mall_a_1", "mall_a_2", "mall_a_3", "mall_a_4", "mall_a_5", "mall_a_6", "mall_a_7", "mall_a_8", "mall_a_9",
+        "strip_mall",
+        "megastore_0_0_0", "megastore_1_0_0", "megastore_2_0_0", "megastore_0_1_0", "megastore_1_1_0", "megastore_2_1_0", "megastore_0_2_0", "megastore_1_2_0", "megastore_2_2_0"
+      ],
+      "message": "You add roads and local malls, strip malls and megastores to your map."
+    }
+  },
+  {
+    "id": "furniturebrochure",
+    "copy-from": "abstractmap",
+    "type": "GENERIC",
+    "name": { "str": "furniture advertising brochure" },
+    "description": "This is a small brochure advertising for local home improvement stores and with an address below each entry. Using it will add roads an local furniture, home improvement and gardening stores to your map.",
+    "color": "light_blue",
+    "use_action": {
+      "type": "reveal_map",
+      "radius": 90,
+      "terrain": [
+        "bridge",
+        "road",
+        "furniture",
+        "home_improvement",
+        "hdwr_large_1_2_0", "hdwr_large_0_2_0", "hdwr_large_1_1_0", "hdwr_large_0_1_0", "hdwr_large_1_0_0", "hdwr_large_0_0_0",
+        { "om_terrain": "s_gardening", "om_terrain_match_type": "TYPE" }
+      ],
+      "message": "You add roads and local furniture, home improvement and gardening stores to your map."
+    }
+  },
+  {
+    "id": "hardwarebrochure",
+    "copy-from": "abstractmap",
+    "type": "GENERIC",
+    "name": { "str": "hardware advertising brochure" },
```

Comment:
I would transform this into a technology brochure, also including internet cafes and maybe arcades, but this is just an idea.

Reply:
Thats a very good idea!
This might be a stupid question, but should I close the Pull Request, do the change and then create another one?

Reply:
Pushing new changes to the branch this PR is on will cause the PR to automatically add those changes. There is no need to close and resubmit.

Reply:
Thanks alot! I'm still kinda new to this and I massively appreciate the help and feedback!